---
title: "Stops making sense: translational trade-offs and stop codon reassignment"

authors:
- L. J. Johnson
- admin
- C. P. Lichtenstein
- G. S. Elgar
- R. A. Nichols
- P. D. Polly
- S. C. Le Comber

date: "2011-07-01T00:00:00Z"
doi: "10.1186/1471-2148-11-227"

publication_types: ["2"]
publication: In *BMC Evolutionary Biology* 11\:227
#publication_short: In *ICW*
abstract: >
  Efficient gene expression involves a trade-off between (i) premature termination of protein synthesis; and (ii) readthrough, where the ribosome fails to dissociate at the terminal stop. Sense codons that are similar in sequence to stop codons are more susceptible to nonsense mutation, and are also likely to be more susceptible to transcriptional or translational errors causing premature termination. We therefore expect this trade-off to be influenced by the number of stop codons in the genetic code. Although genetic codes are highly constrained, stop codon number appears to be their most volatile feature. In the human genome, codons readily mutable to stops are underrepresented in coding sequences. We construct a simple mathematical model based on the relative likelihoods of premature termination and readthrough. When readthrough occurs, the resultant protein has a tail of amino acid residues incorrectly added to the C-terminus. Our results depend strongly on the number of stop codons in the genetic code. When the code has more stop codons, premature termination is relatively more likely, particularly for longer genes. When the code has fewer stop codons, the length of the tail added by readthrough will, on average, be longer, and thus more deleterious. Comparative analysis of taxa with a range of stop codon numbers suggests that genomes whose code includes more stop codons have shorter coding sequences. We suggest that the differing trade-offs presented by alternative genetic codes may result in differences in genome structure. More speculatively, multiple stop codons may mitigate readthrough, counteracting the disadvantage of a higher rate of nonsense mutation. This could help explain the puzzling overrepresentation of stop codons in the canonical genetic code and most variants.

tags: []

featured: false

url_pdf: 'publication_pdfs/JohnsonCottonLichtensteinElgarNicholsPollyLeComber_2011_StopsMakingSenseTranslationalTradeOffsAndStopCodonReassignment_BMCEvolutionaryBiology.pdf'
url_code: ''
url_dataset: ''
url_poster: ''
url_project: ''
url_slides: ''
url_source: ''
url_video: ''
links:
- icon: open-access
  icon_pack: ai
  name: publisher
  url: https://bmcecolevol.biomedcentral.com/articles/10.1186/1471-2148-11-227
- icon: male
  icon_pack: fas
  name: Steve Le Comber 
  url: https://zslpublications.onlinelibrary.wiley.com/doi/full/10.1111/jzo.12758
- icon: female
  icon_pack: fas
  name: Louise Johnson
  url: https://www.reading.ac.uk/biologicalsciences/about/staff/l-j-johnson.aspx

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
image:
caption: ''
focal_point: ""
preview_only: false

projects: []
#- example woudl ref. project in contents/project/example/index.md

slides: ""
---
